mteb/tasks/Retrieval/eng/LEMBNarrativeQARetrieval.py

```diff
@@ -0,0 +1,67 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNarrativeQARetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="LEMBNarrativeQARetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "narrativeqa",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("narrativeqa subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=["written"],
+        domains=["Fiction", "Non-fiction"],
+        task_subtypes=["Article retrieval"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators="derived",
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples={_EVAL_SPLIT: 10449},
+        avg_character_length=None,
```

Comment:
you can use the `calculate_metadata_metrics` from abstask retrieval for this.

Reply:
I see. Actually, what confuses me is the document saying that for retrieval task, 'this should be the average character length of the query-document pairs'. What's the average character length of a *pair* ? should I report a pair of average length, or the average length of the concatenation of query and document?

Reply:
I agree it isn't too clear, which is why we specified the function (to at least make it consistent).